src/vsp/target.cpp

```diff
@@ -12,6 +12,10 @@
 
 namespace vsp {
 
+static const char* wp_type_str(watchpoint_type type) {
+    return (type == WP_READ) ? "r" : (type == WP_WRITE) ? "w" : "rw";
```

Comment:
```
switch (type) {
case WP_READ:
    return "r";
case WP_WRITE:
    return "w";
case WP_ACCESS:
default:
    return "rw";
}
``` 

Also changing the return type to `static string wp_type_str()` avoids the cast in mkwp.